### PR TITLE
docs: add information about the build dockerfiles

### PR DESCRIPTION
**Pull request by sxa**
Split out from https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1974

Signed-off-by: Stewart X Addison <sxa@redhat.com>

##### Checklist

- [x] commit message has one of the [standard prefixes](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md#commit-messages)
- [x] [FAQ.md](https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptopenjdk.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptopenjdk.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -77,6 +77,29 @@ ssh-add
 and if using the `-b` option, ensure that your user has access to `sudo`
 without a password to the `root` account (often done by adding it to the `wheel` group)
 
+## What about the builds that use the `dockerBuild` tag?
+
+In addition to the static build machines which we have, ther are also
```
Comment (Willsparker) [inline FAQ.md:82]: `ther` --> `there`

File: FAQ.md [hunk 2]
```diff
@@ -77,6 +77,29 @@ ssh-add
...
+Dockerfiles that are used to build the base images that our build farm uses
+for running docker based builds on some of our platforms - this is what we
+have at the moment:
+
+| Dockerfile | Image | Platforms  | Where is this built? | In use?
+|---|---|---|---|
```
Comment (Willsparker) [inline FAQ.md:88]: Rich text isn't displaying a table; needs another `---|`

File: FAQ.md [hunk 3]
```diff
@@ -77,6 +77,29 @@ ssh-add
...
+| [Centos7](./ansible/Dockerfile.CentOS7) | [`adoptopenjdk/centos7_build_image`](https://hub.docker.com/r/adoptopenjdk/centos7_build_image) | linux/amd64, linux/arm64 | [Travis](.travis.yml) | Yes
+| [Centos6](./ansible/Dockerfile.CentOS6) | [`adoptopenjdk/centos6_build_image`](https://hub.docker.com/r/adoptopenjdk/centos6_build_image)| linux/amd64 | [GH Actions](.github/workflows/build.yml) | Yes
+| [Alpine3](./ansible/Dockerfile.Alpine3) | [`adoptopenjdk/alpine3_build_image`](https://hub.docker.com/r/adoptopenjdk/alpine3_build_image) | linux/amd64 | [GH Actions](.github/workflows/build.yml) | Yes
+| [Centos7](./ansible/Dockerfile.CentOS7) | [`adoptopenjdk/centos7_build_image`](https://hub.docker.com/r/adoptopenjdk/centos7_build_image) | linux/amd64, linux/arm64 | [Travis](.travis.yml) | No
```
Comment (Willsparker) [inline FAQ.md:92]: You've got Centos7 twice in the table - though one of them is apparently not in use ..?

**Review by Willsparker — CHANGES_REQUESTED** [review]

**Comment by sxa** [comment]
All comments addressed

**Review by Willsparker — APPROVED** [review]
LGTM. Thanks

**Comment by sxa** [comment]
Tagging @AdamBrousseau FYI since he had commented on the original PR

**Review by AdamBrousseau — APPROVED** [review]
Lgtm, thanks Stewart.

**Pull request merged**